File: app/src/main/java/net/programmierecke/radiodroid2/station/live/metadata/lastfm/data/Artist.java

package net.programmierecke.radiodroid2.station.live.metadata.lastfm.data;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Artist {

    @SerializedName("name")
    @Expose
    private String name;
    @SerializedName("mbid")
    @Expose
    private String mbid;
    @SerializedName("url")
    @Expose
    private String url;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getMbid() {
        return mbid;
    }

    public void setMbid(String mbid) {
        this.mbid = mbid;
    }

    public String getUrl() {
        return url;
    }

    public void setUrl(String url) {
        this.url = url;
    }

}

File: app/src/main/java/net/programmierecke/radiodroid2/station/live/metadata/lastfm/data/Artist.kt
package net.programmierecke.radiodroid2.station.live.metadata.lastfm.data

import com.google.gson.annotations.SerializedName

data class Artist(
        @SerializedName("name")
        val name: String? = null,

        @SerializedName("mbid")
        val mbid: String? = null,

        @SerializedName("url")
        val url: String? = null
)